frontend/packages/console-shared/src/components/getting-started/GettingStartedExpandableGrid.scss

```diff
@@ -0,0 +1,60 @@
+.ocs-getting-started-expandable-grid {
+    --min-column-width: 220px;
+  
+    // Increase css specificity to override a generic [class*="pf-v5-c-"] rule.
+    &__header.pf-v5-c-card__header {
+      // Use padding sm instead of lg to fix alignment of the KebabToggle action button.
+      padding-right: var(--pf-v5-global--spacer--sm);
+    }
+    &__tooltip {
+      white-space: pre-line;
+    }
+    &__tooltip-icon {
+      padding-left: var(--pf-v5-global--spacer--sm);
```

Comment:
When you change this from padding to margin the tooltip changes from this;

![image](https://github.com/openshift/console/assets/139310/2af46746-3998-4c2f-bc6d-3df44695d39f)

to this:

![image](https://github.com/openshift/console/assets/139310/c7486e65-51f9-4760-9a44-09ddda468497)

```suggestion
      margin-left: var(--pf-v5-global--spacer--sm);
```